Makefile

```diff
@@ -0,0 +1,38 @@
+DOCKER_IMG = klakegg/hugo:ext-alpine
+DRAFT_ARGS = --buildDrafts --buildFuture  --buildExpired
+
+# Keep these consistent with the values in config.yaml and layouts/index.redirects:
+NEXTv=v2
+LATESTv=v2
+
+HTMLTEST_DIR=tmp
+HTMLTEST?=htmltest # Specify as make arg if different
+# Use $(HTMLTEST) in PATH, if available; otherwise, we'll get a copy
+ifeq (, $(shell command -v $(HTMLTEST)))
+override HTMLTEST=$(HTMLTEST_DIR)/bin/htmltest
+endif
+
+docker-serve:
+	docker run --rm -it -v $(PWD):/src -p 1313:1313 $(DOCKER_IMG) server $(DRAFT_ARGS)
```

Comment:
Consider dropping docker support (I thought that that is what we had done elsewhere).